DATA/production/qc-workflow.sh

```diff
@@ -22,6 +22,7 @@ if [[ -z $QC_JSON_FROM_OUTSIDE ]]; then
     [[ -z "$QC_JSON_PHS" ]] && QC_JSON_PHS=consul://o2/components/qc/ANY/any/phos-raw-clusters-epn
     [[ -z "$QC_JSON_PRIMVTX" ]] && QC_JSON_PRIMVTX=/home/shahoian/jsons/vertexing-qc.json
     [[ -z "$QC_JSON_GLOBAL" ]] && QC_JSON_GLOBAL=$O2DPG_ROOT/DATA/production/qc-sync/qc-global.json
+    [[ -z "$QC_JSON_TOF" ]] && [[ ! -z "$TOF_SOURCES" ]] && QC_JSON_TOF_MATCH=$O2DPG_ROOT/DATA/production/qc-sync/itstpctof.json
```

Comment:
```suggestion
    [[ -z "$QC_JSON_TOF_MATCH" ]] && [[ ! -z "$TOF_SOURCES" ]] && QC_JSON_TOF_MATCH=$O2DPG_ROOT/DATA/production/qc-sync/itstpctof.json
```
don't you need this instead (similarly in other lines)?

Reply:
you are right, I'll fix it. Thanks